### PR TITLE
[4.0] Fix nested tabs not working in com_config

### PR DESCRIPTION
**Pull request by C-Lodder**
Pull Request for Issue #27188

### Summary of Changes

Fix the com_config view so that nested fieldsets are displayed properly.

### Testing Instructions

1. Go to `administrator/index.php?option=com_config`
2. Select "Articles" from the sidebar menu
3. Go to the "Integrations" tab

### Expected result
72211043-31124b80-34bc-11ea-86a9-672dce975e65

### Actual result
72211050-3ff8fe00-34bc-11ea-9989-62f4b731f379

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 17df359ae18b59058c9010d37eb17974ef7bf402<br /><br />Thanks the functionality has now been restored<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27507">issues.joomla.org/tracker/joomla-cms/27507</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 17df359ae18b59058c9010d37eb17974ef7bf402<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27507">issues.joomla.org/tracker/joomla-cms/27507</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27507">issues.joomla.org/tracker/joomla-cms/27507</a>.</sub>

**Comment by Quy** [comment]
@C-Lodder Please check the Permissions tab as the table is indented.

File: administrator/components/com_config/tmpl/component/default.php [hunk 1]
```diff
@@ -74,34 +73,65 @@
 					<?php endif; ?>
 
 					<?php $label = empty($fieldSet->label) ? 'COM_CONFIG_' . $name . '_FIELDSET_LABEL' : $fieldSet->label; ?>
-					<?php $hasChildren = $xml->xpath('//fieldset[@name="' . $name . '"]/fieldset'); ?>
-
-					<?php echo HTMLHelper::_('uitab.addTab', 'configTabs', $name, Text::_($label)); ?>
 
+					<?php if (!$isGrandchild && $hasParent) : ?>
+						<fieldset id="fieldset-<?php echo $this->escape($name); ?>" class="options-grid-form options-grid-form-full">
```
Comment (Quy) [inline administrator/components/com_config/tmpl/component/default.php]:
```suggestion
						<fieldset id="fieldset-<?php echo $this->escape($name); ?>" class="options-grid-form options-grid-form-full options-menu">
```

File: administrator/components/com_config/tmpl/component/default.php [hunk 2]
```diff
@@ -74,34 +73,65 @@
...
+
+						<?php endif; ?>
+
+						<?php echo HTMLHelper::_('uitab.addTab', 'configTabs', $name, Text::_($label)); ?>
+
+						<?php $opentab = 1; ?>
+
+						<?php if (!$hasChildren) : ?>
+
+						<fieldset id="fieldset-<?php echo $this->escape($name); ?>" class="options-grid-form options-grid-form-full">
```
Comment (Quy) [inline administrator/components/com_config/tmpl/component/default.php]:
```suggestion
						<fieldset id="fieldset-<?php echo $this->escape($name); ?>" class="options-grid-form options-grid-form-full options-menu">
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 717a2344cda904cc34d7fe6fa2250b31b5737314<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27507">issues.joomla.org/tracker/joomla-cms/27507</a>.</sub>

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 717a2344cda904cc34d7fe6fa2250b31b5737314<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27507">issues.joomla.org/tracker/joomla-cms/27507</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27507">issues.joomla.org/tracker/joomla-cms/27507</a>.</sub>

**Comment by wilsonge** [comment]
Thanking you muchly kind sir ;)

**Pull request merged**